### PR TITLE
Add optional custom feedback page

### PR DESCRIPTION
**Pull request by wivern**
for #544 
Add ability to customize feedback url

### CODE REVIEW

**Comment by fdefalco** [comment]
Could you explain what the importer.js file is being used for in the feedback page?

File: js/components/importer.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+define(['knockout', 'text!./importer.html', 'atlas-state', 'databindings'], function (ko, view, sharedState) {
```
Comment (anthonysena) [inline js/components/importer.js]: Per our skype conversation, I believe this file is not needed and can be removed. Can you remove this? We should then be able to get this merged.

**Comment by anthonysena** [comment]
@wivern - made a few small changes here so please review https://github.com/OHDSI/Atlas/pull/602/commits/c583e1c3f44eaac8567afad7195d377df0622d93. Please resolve the conflicts and then we should be set to merge this into master. Thanks!

**Comment by anthonysena** [comment]
@wivern would it be easier to put these changes into another branch based off the current master and file a new PR that we can merge in?

**Pull request merged**